### PR TITLE
Tests For Restarts and Fixes

### PR DESCRIPTION
**Pull request by gene-redpanda**
Contains a fix for a few toggle bugs (set to one thing then set to another) that were causing issues and support for testing the restart detection process.

### CODE REVIEW

File: roles/redpanda_broker/Makefile [hunk 1]
```diff
@@ -1,22 +1,27 @@
-.PHONY: test reqs do
-
 PYTHON ?= python
 PIP ?= pip
 
 DIR := tests
+### TODO revert to original scripts
```
Comment (r-vasquez) [inline roles/redpanda_broker/Makefile:5]: Is this a leftover?

File: roles/redpanda_broker/defaults/main.yml [hunk 2]
```diff
@@ -66,3 +66,5 @@ rp_repo_signing_deb: "deb [signed-by=/usr/share/keyrings/redpanda-redpanda-archi
 rp_repo_signing_deb_unstable: "deb [signed-by=/usr/share/keyrings/redpanda-redpanda-unstable-archive-keyring.gpg] {{ redpanda_base_url }}/E4xN1tVe3Xy60GTx/redpanda-unstable/deb/{{ansible_distribution | lower}} {{ ansible_distribution_release | lower }} main"
 rp_repo_signing_src_deb: "deb-src [signed-by=/usr/share/keyrings/redpanda-redpanda-archive-keyring.gpg] {{ redpanda_base_url }}/public/redpanda/deb/{{ansible_distribution | lower}} {{ ansible_distribution_release | lower }} main"
 rp_repo_signing_src_deb_unstable: "deb-src [signed-by=/usr/share/keyrings/redpanda-redpanda-unstable-archive-keyring.gpg] {{ redpanda_base_url }}/E4xN1tVe3Xy60GTx/redpanda-unstable/deb/{{ansible_distribution | lower}} {{ ansible_distribution_release | lower }} main"
+
+redpanda_broker_no_log: true
```
Comment (r-vasquez) [inline roles/redpanda_broker/defaults/main.yml:70]: [question] Why is this toggle needed?

**Review by r-vasquez — COMMENTED** [review]
LGTM, just a small question and a TODO (possibly leftover from testing).

Another question: what was the perm issue? why is the need to change it and what kind of users were affected by it?

_(if possible, add this info in the PR description or commit for future reference)_

File: roles/redpanda_broker/defaults/main.yml (hunk 2 above)
Comment (gene-redpanda) [reply roles/redpanda_broker/defaults/main.yml:70]: no_log causes ansible to not emit sensitive data so we put that everywhere by default. Making it a toggle in this manner makes debugging a lot easier.

File: roles/redpanda_broker/Makefile (hunk 1 above)
Comment (gene-redpanda) [reply roles/redpanda_broker/Makefile:5]: Yeah, leftover from me excluding the other tests I'll clean it up

**Comment by gene-redpanda** [comment]
> LGTM, just a small question and a TODO (possibly leftover from testing).
> 
> Another question: what was the perm issue? why is the need to change it and what kind of users were affected by it?
> 
> _(if possible, add this info in the PR description or commit for future reference)_

Meant to merge the that commit into the "flipping between root and rp" commit which explains the situation in the commit message. Resolved!

**Review by r-vasquez — APPROVED** [review]
LGTM!

**Pull request merged**